src/batch-client/index.ts

```diff
@@ -84,8 +84,14 @@ function normalizeMessage(
 	message: { [key: string]: any },
 	zimbraOrigin?: string
 ) {
+	const normalizedMessage = normalize(MessageInfo)(message);
+	normalizedMessage.attributes = mapValuesDeep(
```

Comment:
please check if attributes are present then only do this processing

Reply:
Done!